packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -505,6 +515,9 @@ export class GaussianSplattingMesh extends Mesh {
             return new RawTexture(data, width, height, format, this._scene, false, false, Constants.TEXTURE_BILINEAR_SAMPLINGMODE, Constants.TEXTURETYPE_FLOAT);
         };
 
+        const updateTextureFromData = (texture: BaseTexture, data: Float32Array, width: number, height: number) => {
+            (this.getEngine() as ThinEngine).updateTextureData(texture.getInternalTexture()!, data, 0, 0, width, height, 0, 0, false);

```

Comment:
I guess `updateTextureData` should be in `AbstractEngine`? Looks like `WebGPUEngine` implements it as well?